.github/workflows/pacakage.yml

```
@@ -0,0 +1,40 @@
+  ENV_VAR: test
+
+jobs:
+  Package-test:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+      
+    - name: Setup JDK 17
+      uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)